manifests/w/Wings3D/Wings3D/2.3/Wings3D.Wings3D.installer.yaml

```diff
@@ -0,0 +1,37 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Wings3D.Wings3D
+PackageVersion: '2.3'
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Custom: /NORESTART
+UpgradeBehavior: install
+ReleaseDate: 2023-10-09
+Installers:
+- Architecture: x86
+  InstallerUrl: https://sourceforge.net/projects/wings/files/wings/2.3/wings-2.3.exe/download
+  InstallerSha256: EE9078DAD767DE4EAF0C9DFD85C93753A6C3FAD822A83A66F40730EC0BB29D29
+  Dependencies:
+    PackageDependencies:
+    - # NOTE: Dependency version has been updated!
+      PackageIdentifier: Microsoft.VCRedist.2015+.x86
+      MinimumVersion: 14.29.30139.0
+- Architecture: x64
+  InstallerUrl: https://sourceforge.net/projects/wings/files/wings/2.3/wings-x64-2.3.exe/download
+  InstallerSha256: 5E425B2E91DD03DEB88697E0050EE6B3C1C31292B4A65D58FCDBFA6739270394
+  Dependencies:
+    PackageDependencies:
+    - # NOTE: Dependency version has been updated!
+      PackageIdentifier: Microsoft.VCRedist.2015+.x64
+      MinimumVersion: 14.29.30139.0
```

Comment:
Dependency of this package version has been upgraded, since it's compiled by MSVC 14 now.